packages/core/src/Stores/traders-hub-store.js

```diff
@@ -395,6 +397,12 @@ export default class TradersHubStore extends BaseStore {
         this.setCombinedCFDMT5Accounts();
     }
 
+    get restricted_countries_filter_content() {
```

Comment:
@aizad-deriv Also this can be a hook as it's only using client store data, But you are using this method in `CFDs_restricted_countries` in this store which we can't use hook here :(

What do you guys think? Should we create a hook for it or we can ignore it for now? 🤔

Reply:
Can we ignore it for now 😩 I need this to be merged because it is a blocking another card of mine 😞.
I'll address it there 😃 👍🏼

Reply:
@aizad-deriv Sure man let's move forward 🙇🏻 Please ignore this comment 🙏🏻